sources/PTYTask.m

```diff
@@ -43,6 +43,8 @@
                 struct winsize* win,
                 int width,
                 int height,
+                int pixelWidth,
```

Comment:
If you're feeling ambitious, yes please! This code is old and crufty and I like to clean it up as I go, but I won't hold up this PR for that.